bedrock/contentful/api.py

```diff
@@ -85,17 +99,15 @@ def get_client(raw_mode=False):
             api_url=settings.CONTENTFUL_SPACE_API,
             raw_mode=raw_mode,
             content_type_cache=False,
+            timeout_s=settings.CONTENTFUL_API_TIMEOUT,
         )
 
     return client
 
 
 def contentful_locale(locale):
     """Returns the Contentful locale for the Bedrock locale"""
-    if locale.startswith("es-"):
-        return "es"
```

Comment:
@pmac  I pulled this because it no longer seemed to match the Contentful or Bedock locales (both have es-MX etc) -- do you recall why this was added originally?

Reply:
I don't. Must have been from an earlier experiment with Contentful l10n. Good catch.